src/App.js

```diff
@@ -1,16 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  console.log(chatMessages);
+  const [chatsMessages, setChatState] = useState(chatMessages);
+  const changeLike = (id) => {
+    const newData = chatsMessages.map((chat) => {
+      if (chat.id === id) {
+        chat.liked = !chat.liked;
+        return chat;
+      }
+      return chat;
```

Comment:
Since you have `return chat;` inside and outside the if statement, but you need to return chat in both cases, you can delete line 13 and let line 15 do the returning (which will happen regardless of whether line 12 executes or not